README.md

```diff
@@ -26,6 +26,8 @@ of the URI as the path key. If the image is already downloaded, it will be rende
 CachedImage is a direct wrapper of [react-native Image](https://facebook.github.io/react-native/docs/image)
 and matches it's API. As such, all of its props are available as props to CachedImage. Styles are also passed down.
 
+You can assign your own style to Image/ImageBackground view as shown above, and it's a mandatory props, without this Image/ImageBackground won't be displayed. The style should contain image height and width.
```

Comment:
Last thing: let's remove this readme addition because it is innacurate. Styles work the same as with the image component.